core/storage/exploration/gae_models.py

```diff
@@ -386,8 +487,16 @@ def get_top_rated(cls, limit):
 
     @classmethod
     def get_private_at_least_viewable(cls, user_id):
-        """Returns an iterable with private exp summaries that are at least
+        """Fetches private exp summaries that are at least
```

Comment:
Add words from the following line to the end of this one so that it still stays within the 80-char limit.